databases/week1/week1.sql

```diff
@@ -0,0 +1,59 @@
+-- Active: 1719823579315@@0.0.0.0@3306@hyfdatabase
+
+
+-- count the number of tasks (35)
+select count (*) from task;
+
+--how many tasks in the task table do not have a valid due date (8)
+select count (*) as invalid_due_date
+from task
+where due_date IS NULL;
+
+-- all the tasks that are marked as done(there are 12)
+
+select t.title as done_tasks
+from task t
+join status s on t.status_id = s.id
+where s.name ="Done";
+
+--all the tasks that are not marked as done (23)
+select t.*
+from task t
+join status s on t.status_id = s.id
+where s.name = "Not started" or s.name = "In progress";
```

Comment:
Instead of writting all the possible options besides "done", you could just write WHERE NOT status.name = "done". Imagine you could have more than 3 possible options.